app/views/course/assessment/assessments/_form.html.slim

```diff
@@ -8,7 +8,7 @@
   = f.input :start_at
   = f.input :end_at
   = f.input :bonus_end_at
-  = f.input :draft
+  = f.input :draft, disabled: @assessment.questions.empty?
```

Comment:
Sounds good!

Added the custom message to draft and display it only if it is not new.